chapters/classes.tex

```diff
@@ -409,14 +409,18 @@ \subsection{Conditional Component Declaration}\label{conditional-component-decla
 \cref{interface-compatibility-or-subtyping}).
 
 If the \lstinline!Boolean! expression is false the component (including its modifier) is removed from the flattened DAE, and connections to/from the component are removed.
-A component declared with a condition-attribute can only be modified and/or used in connections.
+Such a component can only be modified, used in connections, and/or used in a modifier of another conditional component with a false condition.
+
+There are no restrictions on the component if the \lstinline!Boolean! expression is true.
```

Comment:
Don't we usually highlight true and false in text like that? It is highlighted in the other parts of this chapter. I also started #3561 to fix false in this section.
```suggestion
There are no restrictions on the component if the \lstinline!Boolean! expression is \lstinline!true!.
```